app/controllers/application_controller.rb

```diff
@@ -29,6 +31,14 @@ class ApplicationController < ActionController::Base
     end
   end
 
+  def show_max_size_warning?
+    ENV['SHOW_LOOMIO_ORG_MARKETING'] and @group and @group.approaching_max_size? and !@group.max_size_reached?
```

Comment:
Ooops, looks like this might have been pulled out of the ruby spec recently. Ignore!